File: Servidor/AtHomeTFGCarlosHernandez/src/main/java/org/tfg/athometfgcarloshernandez/spring/model/response/PantallaInmueblesResponseDTO.java
package org.tfg.athometfgcarloshernandez.spring.model.response;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.tfg.athometfgcarloshernandez.spring.model.CajonDTO;
import org.tfg.athometfgcarloshernandez.spring.model.HabitacionDTO;
import org.tfg.athometfgcarloshernandez.spring.model.MuebleDTO;

import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class PantallaInmueblesResponseDTO {
    private List<HabitacionDTO> habitaciones;
    private List<MuebleDTO> muebles;
    private List<CajonDTO> cajones;
}

File: Cliente/AtHomeTFGAndroidCarlosHernandez/app/src/main/java/com/example/athometfgandroidcarloshernandez/data/model/response/PantallaInmueblesResponseDTO.kt
package com.example.athometfgandroidcarloshernandez.data.model.response

import com.example.athometfgandroidcarloshernandez.data.model.CajonDTO
import com.example.athometfgandroidcarloshernandez.data.model.HabitacionDTO
import com.example.athometfgandroidcarloshernandez.data.model.MuebleDTO

data class PantallaInmueblesResponseDTO (
    val habitaciones: List<HabitacionDTO> = emptyList(),
    val muebles: List<MuebleDTO> = emptyList(),
    val cajones: List<CajonDTO> = emptyList()
)